README.md

```diff
@@ -3,287 +3,106 @@
-
-Download the latest 1.1.x point release of [ElasticSearch](http://www.elasticsearch.org/download/), and extract it to a directory of your choice. Once extracted, you can start it by running the following:
-
-```
-cd my-elasticsearch-dir
-bin/elasticsearch -f
-```
-
-To start it in the background, you can omit the `-f` parameter.
-
-ElasticSearch powers the full-text search functionality of OAE.
-
-#### RabbitMQ
-
-To install RabbitMQ, please follow the instructions on the [RabbitMQ download page](http://www.rabbitmq.com/download.html). Once completed, you should be able to start RabbitMQ by running the following:
-
-```
-rabbitmq-server
-```
-
-To start it in the background, you can run: `rabbitmq-server -detached`
-
-RabbitMQ powers the asynchronous task-queue function in Hilary. It allows heavier "background" tasks such as activity processing, search indexing and preview processing to be off-loaded to specialized clusters of servers. Though, in a development environment you don't need to worry about specialized clusters, your development machine will do just fine out-of-the-box.
-
-#### GraphicsMagick
-
-GraphicsMagick installation instructions can be found on their [README page](http://www.graphicsmagick.org/README.html), however for *nix OS' it is typically available in the package manager of your choice (e.g., `brew install graphicsmagick`).
-When installing GraphicsMagick manually, make sure you have at least [libpng](http://www.libpng.org/pub/png/libpng.html), [libjpeg](http://libjpeg.sourceforge.net) and [Ghostscript](http://www.ghostscript.com) installed.
-
-GraphicsMagick provides the ability to crop and resize profile pictures, and is required to run Hilary.
-
-#### Preview Processor (optional)
-
-The preview processor is not a requirement to run Hilary, but it certainly makes things look wonderful. It takes care of producing previews of content items for the UI (e.g., splitting PDFs into pages, cropping / resizing uploaded images). There are a few dependencies needed only if you are planning to run the preview processor:
-
-##### pdf2htmlEX (only if preview processor is desired)
-
-Download and install [pdf2htmlEX](https://github.com/coolwanglu/pdf2htmlEX) **0.11 or later**. This dependency takes care of converting a pdf file to a set of HTML files.
-
-On Ubuntu this can be installed by running:
-```
-sudo add-apt-repository ppa:coolwanglu/pdf2htmlex
-sudo apt-get update
-sudo apt-get install pdf2htmlEX
-```
-
-##### LibreOffice (only if preview processor is desired)
-
-Download and install [LibreOffice](http://www.libreoffice.org/download/). This dependency takes care of converting Microsoft Office files to PDFs so they may be further split into previews by PDFTK.
+## Installation
 
-##### pdftotext (only if preview processor is desired)
+If you're looking to install the OAE project manually, check out [this page](https://github.com/brecke/Hilary/wiki/Manual-installation-&-setup) and then go the the [Setup section](#setup) below.
```

Comment:
This Wiki page will need to be created in the oaeproject Wiki and the path should be updated to point to it.